src/libraries/System.IO.Pipes.AccessControl/src/Resources/Strings.resx

```diff
@@ -57,6 +57,15 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="ArgumentOutOfRange_NeedValidPipeAccessRights" xml:space="preserve">
```

Comment:
What was the reason for the addition of these resource strings? I don't see src changes in this PR, which is why I ask.